XIVSlothCombo/Combos/PvE/SAM.cs

```diff
@@ -482,6 +495,15 @@ protected override uint Invoke(uint actionID, uint lastComboMove, float comboTim
 
                                     if (IsEnabled(CustomComboPreset.SAM_ST_GekkoCombo_CDs_Shoha) && Shoha.LevelChecked() && gauge.MeditationStacks == 3)
                                         return Shoha;
+
+                                    // healing - please move if not appropriate this high priority
```

Comment:
These are nested under `SAM_ST_GekkoCombo_CDs` which means your customcombopreset will need to be adjusted accordingly

Reply:
Do you think that's the best way to do it? I was fretting over doing that, which would make it unique among all the implementations or having it's own separate canWeave location for it. But I must confess, the Sam code is a bit too big brain for me to figure out quite where that would be

Reply:
It's either that or move it somewhere else. Users will need to have `SAM_ST_GekkoCombo_CDs` on as well then and if it's not nested then there's no indication that it's required to be on for your healing feature to work.